.github/workflows/test.yml

```diff
@@ -0,0 +1,59 @@
+# Heavily based on https://joelhooks.com/jest-and-github-actions
+name: Tests CI
+
+on:
+  pull_request:
+    paths-ignore:
+        - 'README.md'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Reconfigure git to use HTTPS authentication
+        uses: GuillaumeFalourd/SSH-to-HTTPS@v1
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '17'
+
+      - name: Install
+        run: |
+          npm ci
+
+      - name: Run ESLint
+        run: npm run lint
+
+      - name: Run Jest tests
+        run: npm run test:ci
```

Comment:
This script is currently not in the `package.json`, can you please remove this step? We'll add it once we'll actually add tests to the repository

Reply:
@robbertvancaem, Added test for just subtitle plugin, can I keep this step now?

Reply:
Yes, that should be no problem